src/apps/mux/helpers/mux.lev-trades.balance-helper.ts

```diff
@@ -0,0 +1,97 @@
+import { Inject, Injectable } from '@nestjs/common';
+import Axios, { AxiosInstance } from 'axios';
+import _ from 'lodash';
+
+import { drillBalance } from '~app-toolkit';
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { formatMetaBaseData, getCollateralTokensByNetwork, getMarketTokensByNetwork } from '~apps/mux/helpers/common';
+import { Network, NETWORK_IDS } from '~types/network.interface';
+
+import { MuxContractFactory } from '../contracts';
+import MUX_DEFINITION from '../mux.definition';
+
+export type MuxLevTradesContractPositionDataProps = {
+  collateralTokenSymbol: string;
+  marketTokenSymbol: string;
+  isLong: boolean;
+};
+
+type GetLevTradesContractPositionHelperParams = {
+  network: Network;
+  address: string;
+};
+
+@Injectable()
+export class MuxLevTradesBalanceHelper {
+  private axios: AxiosInstance = Axios.create({
+    baseURL: 'https://app.mux.network',
+  });
+
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
+    @Inject(MuxContractFactory) private readonly contractFactory: MuxContractFactory,
+  ) {}
+
+  async getBalance({ address, network }: GetLevTradesContractPositionHelperParams) {
+    const parameter = `[{"type":"id","value":"${address.toLowerCase()}","target":["variable",["template-tag","trader"]],"id":"14ef4983"}]`
+    const pathUrl = `/metabase/api/public/dashboard/5fe8cebe-328e-4334-8fb3-52c98fc8ef71/dashcard/128/card/92?parameters=${encodeURIComponent(
+      parameter,
+    )}`;
+    const { data } = await this.axios.get(pathUrl);
```

Comment:
I suggest you fetch balances calling the chain directly

Reply:
This is the interface used on our project to get user balances (https://app.mux.network/#/leaderboard/detail/0xaea90572acd703fa8d6b72c2c2afbc83ed0e7272?chainId=42161). We want to avoid doing duplicate work as much as possible.

Reply:
Done! Please check it again.